src/site/playground/src/App.tsx

```diff
@@ -16,27 +16,33 @@ const darkTheme = createTheme({
 function App() {
   return (
     <ThemeProvider theme={darkTheme}>
-      <Box height="100vh" display="flex" flexDirection="column" rowGap={2}>
+      <Box
+        height={{ sm: "100vh" }}
+        overflow="hidden"
+        display="flex"
+        flexDirection="column"
+        rowGap={1}
+      >
         <Box
           component="header"
           display="flex"
           alignItems="center"
           gap={2}
           paddingBlock={2}
-          paddingInline={2}
-          bgcolor="#333"
+          paddingInline={{ xs: 1, sm: 2 }}
+          borderBottom="1px solid var(--border-primary)"
         >
           <span
             style={{
               display: "flex",
               alignItems: "center",
               gap: "8px",
-              fontSize: "20px",
-              fontWeight: 600,
             }}
           >
-            <img src={wirespecLogo} height={30} alt="Wirespec Logo" />
-            <span> Wirespec Playground </span>
+            <img src={wirespecLogo} height={30} alt="" />
```

Comment:
removed the alt as the logo is decorative and accompanied by text, see https://www.w3.org/WAI/tutorials/images/decorative/